.github/workflows/lint.yml

```diff
@@ -25,6 +25,6 @@ jobs:
           pip install --upgrade pip
           pip install --upgrade flake8
           # Fail on syntax errors, undefined names, and unused imports
-          flake8 . --select=E9,F4,F63,F7,F82,F84 --statistics
```

Comment:
Do you want to leave this lint check out? Personally a fan of it myself.

Reply:
I'm a fan of it too! The only reason I took out `F4` specifically is because (as I understand), it seems like some of our web-app-related files need to have unused imports. For example, `/sparkle/app/__init__.py` has two unused imports at the very bottom. In one of her videos, Diane mentioned why we needed them at the bottom, had to do with circular imports or something. Anyways, though those imports aren't used within the `__init__.py` file itself, they are used by other module that imports `app`

This is all my understanding of things. I would also like to consult with @ajcheon to verify! Perhaps there is a better work-around.